src/brain/issueLabelHandler/route.ts

```diff
@@ -181,7 +140,7 @@ export async function markRouted({
   const owner = payload.repository.owner.login;
   const octokit = await getClient(ClientType.App, owner);
   const labelsToRemove: string[] = [];
-  const labelNames = issue?.labels?.map(label => label.name) || [];
+  const labelNames = issue?.labels?.map((label) => label.name) || [];
```

Comment:
?

Reply:
doesn't do anything, I can definitely change this back